app/src/main/java/com/example/moviedb/ui/genre/GenreViewModel.kt

```diff
@@ -14,23 +14,25 @@ class GenreViewModel : BaseViewModel() {
     private val movieRepository =
         MovieRepository.getInstance(RetrofitInstance.getInstance().create(ApiService::class.java))
 
+    var isExhaust = false
+
+    var currentPage: Int = 0
+
     private val compositeDisposable = CompositeDisposable()
 
     val mutableLiveData = MutableLiveData<MoviesResponse>()
```

Comment:
nếu anh không dùng để binding trực tiếp ra layout thì anh nên viết ntn chứ: 
https://developer.android.com/topic/libraries/architecture/viewmodel#implement
https://www.youtube.com/watch?v=OMcDk2_4LSk&feature=youtu.be&t=65
![image](https://user-images.githubusercontent.com/53286739/73621863-4406de80-466a-11ea-9caf-cfd4337eb002.png)